book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+(ch-content)=
+# Writing Content
+
+As _The Turing Way_ project grows and evolves, we actively welcome and encourage continuous contributions from community members in the form of new chapters and case studies. 
+However, these new contributions need to be consistent with the overall theme, purpose, format, and style of the book's existing content.
+
+Since _The Turing Way_ is written asynchronously by multiple authors around the world, we have created several templates to guide future content contributions in order to preserve the book's consistency as it grows.
+
+This chapter provides a brief overview of these templates. 
```

Comment:
We don't actually talk about the templates much here. It's more about where the files need to be placed in the directory. Eventually, there's also a contribution checklist which again, is not really _templates_.